docker/Dockerfile.custom_k8s_config

```diff
@@ -0,0 +1,18 @@
+# Dockerfile for building an image of scalyr/scalyr-docker-agent with custom configuration
```

Comment:
I thought we were going to have a different image name for the k8 build of the agent, such as `scalyr/scalyr-k8-agent`